src/concurrency/async/async-await.md

```diff
@@ -31,9 +31,6 @@ Key points:
 - Note that this is a simplified example to show the syntax. There is no long
   running operation or any real concurrency in it!
 
-- What is the return type of an async call?
-  - Use `let future: () = async_main(10);` in `main` to see the type.
-
```

Comment:
I think it's still worthwhile to take the step of demonstrating that the return type of an async fn is not the written return type, but actually a future type.

Reply:
I agree that the answer is now less detailed (and less confusing), but I agree with @fw-immunant) that this should stay:
 - it illustrates that this is not a magical exception from the type rules
 - in general, it's a good technique for finding the type of a value, and bears a bit of repeating